### DISCUSSION TITLE
WAN

### DISCUSSION THREAD

**Discussion by cloudandzack007** [OpenDDS/OpenDDS#3984] [Q&A] [answered]
I want to know how to confige OpenDDS at WAN.
My computers locate at different LAN，I want to connect them by OpenDDS.

**Comment by mitza-oci** [comment, marked as answer]
OpenDDS can work on any IP-based network.  Without any other information in your question it's difficult to provide more guidance.

**Reply by cloudandzack007** [reply]
for example，ip address of pc A is 192.168.1.2，ip address of pc B is 172.1.100.8，they all can connect internet，but can not connect each other directly。How to  connect them by opendss at inernet？

**Reply by mitza-oci** [reply]
See the Developer's Guide chapter 15 "Internet-Enabled RTPS"

**Reply by cloudandzack007** [reply]
thank you
